diesel_derives/src/lib.rs

```diff
@@ -411,15 +413,21 @@ pub fn derive_query_id(input: TokenStream) -> TokenStream {
 ///
 /// # Attributes
 ///
-/// ## Optional field attributes:
+/// ## Optional container attributes
+///
+/// * `#[table_name = "some_table_name"]`, specifies the table this type
+/// is insertable into. Requires that `some_table_name` is in scope.
+/// If this attribute is not used, the type name converted to
+/// `snake_case` with an added `s` is used as table name
```

Comment:
We do not accept a `#[table_name]` attribute for `Queryable`.

Reply:
Ops, I thought that was `Insertable`, my mistake, I got confused a bit.
```suggestion
```